nixos/modules/config/xdg/autostart.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+{
+  options = {
+    xdg.autostart.enable = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Whether to install files to support the XDG Autostart specification.
```

Comment:
It would be nice to have a link to the specification `<link xlink:href="https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html">XDG Autostart specification</link>`